coordinator/src/cli.rs

```diff
@@ -64,6 +64,10 @@ pub struct Opts {
         default_value = "16f88cf7d21e6c0f46bcbc983a4e3b19726c6c98858cc31c83551a88fde171c0"
     )]
     oracle_pubkey: String,
+
+    /// The endpoint of the maker.
+    #[clap(long, default_value = "http://localhost:18000")]
```

Comment:
wouldn't it be nicer the maker calls an coordinator api instead of the coordinator? You are anyways only doing the sync in an interval. That could as well also happen on the maker.